package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file should be removed. It is not needed

Reply:
I don't know how that snuck in. I was having a really hard time pushing from visual studio on Friday (if that wasn't already apparent).